apps/src/studio/Item.js

```diff
@@ -293,6 +314,10 @@ Item.prototype.turnToFaceActor = function (targetSpriteIndex) {
   }
 };
 
+Item.prototype.setActivity = function(type) {
```

Comment:
I'd appreciate a comment on this function about what an "activity" is in the context of an item.